sonarts-core/src/runner/sonartsServer.ts

```diff
@@ -33,7 +33,16 @@ const servicesPerTsconfig: Map<string, ts.LanguageService> = new Map();
 
 const EMPTY_ANSWER: { issues: any[] } = { issues: [] };
 
+function logTypeScriptMetaInfo() {
+  const version = require("typescript/package.json").version;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: require() style import is forbidden ([typescript:S3533](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3533))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=issue-460&issues=AWEjs5pa1DTsLFfQjJ3o&open=AWEjs5pa1DTsLFfQjJ3o)

[ISSUE_KEY]: AWEjs5pa1DTsLFfQjJ3o

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED